src/pages/SwapV2/index.tsx

```diff
@@ -564,7 +562,11 @@ export default function Swap({ history }: RouteComponentProps) {
         )}&networkId=${chainId}`
       : window.location.origin + `/swap?networkId=${chainId}`
 
-  const renderTokenInfo = Boolean(isShowTokenInfoSetting && (currencyIn || currencyOut))
+  const renderTokenInfo =
```

Comment:
This should be renamed to shouldRenderTokenInfo as it's a boolean

Reply:
okay, updated.